server/server.go

```diff
@@ -122,7 +123,14 @@ func (s *Server) Start(ctx context.Context) error {
 		}),
 	}
 
-	proofsDbPath := filepath.Join(s.cfg.DataDir, "proofs")
+	proofsDbPath := filepath.Join(s.cfg.DbDir, "proofs")
+	// Temporary migration to move DB to a new location.
+	// TODO(poszu): remove this code after all poets are migrated.
+	oldProofsDbPath := filepath.Join(s.cfg.DataDir, "proofs")
```

Comment:
I think we should also mention this in the release notes, otherwise someone operating their own PoET might miss it.

We should also mention it again when this is removed and that new layout is now expected.

Reply:
Sure, I will add information about the new configuration option to specify a different database directory to release notes.

It defaults to the original behaviour (`DbDir` defaults to `DataDir`). If somebody is unware and runs without specifing the former, then everyting will work as before. The migration code is there only to help make the transition **only once** - it will not migrate if `DbDir` is changed from A to B.